app/templates/components/forms/wizard/sponsors-step.hbs

```diff
@@ -20,7 +21,7 @@
     </div>
   </div>
 
-  {{#if this.data.event.isSponsorsEnabled}}
```

Comment:
this is done because when we toggle wizard pages, sometimes checkbox is pre-checked even if this.data.event.isSponsorsEnabled is false. i have used this.data.sponsors.length and it is working fine.